classwork/advanced-05-classwork.qmd

```diff
@@ -117,68 +112,27 @@ lgbm_bayes_res <-
     initial = init_res,     # <- initial results
     iter = 20,
     param_info = lgbm_param,
+    control = ctrl_bo,
     metrics = reg_metrics
   )
-
-show_best(lgbm_bayes_res, metric = "mae")
-```
-
-Plotting results
-
-```{r}
-autoplot(lgbm_bayes_res, metric = "mae")
-autoplot(lgbm_bayes_res, metric = "mae", type = "parameters")
-autoplot(lgbm_bayes_res, metric = "mae", type = "performance")
 ```
 
-## Your turn
 
-Let's try a different acquisition function: `conf_bound(kappa)`.
+## Your turn {transition="slide-in"}
 
-We'll use the `objective` argument to set it.
+*Let's try a different acquisition function: `conf_bound(kappa)`.*
 
-Choose your own `kappa` value:
+*We'll use the `objective` argument to set it.*
 
-- Larger values will explore the space more.
-- "Large" values are usually less than one.
+*Choose your own `kappa` value:*
 
-```{r}
-# Your code here!
+ - *Larger values will explore the space more.* 
+ - *"Large" values are usually less than one.*
 
-```
+**Bonus points**: Before the optimization is done, press `<esc>` and see what happens.
 
-## Finalize the workflow
 
 ```{r}
-best_param <- select_best(lgbm_bayes_res, metric = "mae")
-
-final_wflow <- 
-  lgbm_wflow %>% 
-  finalize_workflow(best_param)
-
-final_wflow
-```
-
-## The Final Fit
```

Comment:
No more finalization stuff? I guess the focus is more on how the optimization works in general?

I imagine you are cutting a good bit for time since it is only 1 day

Reply:
It's in the slides but we don't have any classwork related to that so I took it out if the classwork file